tracer/test/Datadog.Trace.TestHelpers/MockHttpParser.cs

```diff
@@ -194,85 +193,5 @@ public async Task<bool> IsNewLine()
 
         private class ChunkedEncodingReadContent(Stream stream)
             : StreamContent(new ChunkedEncodingReadStream(stream), length: null);
-
-        private class ChunkedEncodingReadStream(Stream innerStream) : DelegatingStream(innerStream)
```

Comment:
TIL we had a `ChunkedEncodingReadStream` here...

Reply:
We added it to handle the chunked responses we need to _send_ with the tracer flare. But that version was hacky and didn't handle edge cases as it was just for testing!